crypto/src/cortex_m0_reduce25519.s

```diff
@@ -7,6 +7,9 @@
 //
 // Generated and tested with C++ functions in the test subdirectory and on the target.
 //
+// This assumes the program fe25519_reduceTo256Bits_asm with takes in two inputs, 

```

Comment:
"exports" or "exposes" not "assumes". "assumes" implies there is such a function defined somewhere else and this file relies on that function.